src/index.ts

```diff
@@ -34,6 +47,14 @@ import { WebsocketService } from './services/websocket_service';
     const orderBookService = new OrderBookService(connection, meshClient);
     // tslint:disable-next-line:no-unused-expression
     new SRAHttpService(app, orderBookService);
+    // Quote Service
+    const orderStore = new OrderStoreDbAdapter(orderBookService);
+    const orderProvider = new OrderBookServiceOrderProvider(orderStore, orderBookService);
+    const orderBook = new Orderbook(orderProvider, orderStore);
+    const provider = createWeb3Provider(config.ETHEREUM_RPC_URL);
+    // tslint:disable-next-line:no-unused-expression
+    new SwapQuoteHttpService(app, orderBook, provider);
```

Comment:
Can we wrap this in a try {} catch{} so that the mesh logic below can execute in case anything throws?